gravitee-apim-cypress/cypress.json

```diff
@@ -3,7 +3,7 @@
   "baseUrl": "http://localhost:8083",
   "video": false,
   "screenshotOnRunFailure": false,
-  "integrationFolder": "cypress/integration/nrt",
+  "integrationFolder": "cypress/integration/",
```

Comment:
it will also run `bulk` folder which is not a test folder.
move `bulk` folder away from `integration`

Reply:
As discussed, the bulk folder will stay there for now.